packages/inquirer/lib/prompts/input.js

```diff
@@ -89,7 +89,9 @@ class InputPrompt extends Base {
     this.done(state.value);
   }
 
-  onError(state) {
+  onError(state) { 
+    this.rl.line += state.value;
```

Comment:
Should this just be `=`?

Reply:
I'll modify it  as `+=`

Thank you for review.

Reply:
Actually, `+=` `=` are doesnt matter. Because `this.rl.cursor` is set to `0` on error.
But for readability using `+=` is more reasonable i think.

Reply:
I'm not sure it's more readable, what we want is to keep the answer on the line. Having `+=` is confusing because if the line wasn't cleaned and cursor reset to 0, we'd end up duplicating the answer. Without knowing the internal of how readline works, this would be quite confusing.